pkg/version/version.go

```diff
@@ -22,7 +22,8 @@ import (
 	"github.com/blang/semver"
 )
 
-var criContainerdVersion = "UNKNOWN"
+// CRIContainerdVersion is the version of the cri-containerd.
+var CRIContainerdVersion = "UNKNOWN"
```

Comment:
why change the case here, to make this a public var? curious

Reply:
To use it in `server/version.go`